build/pxc-entrypoint.sh

```diff
@@ -254,6 +254,12 @@ else
 
 fi
 
+WSREP_CLUSTER_NAME=$(grep wsrep_cluster_name  ${CFG}| cut -d '=' -f 2| tr -d ' ' )
+if [[ !  ${WSREP_CLUSTER_NAME} || ${WSREP_CLUSTER_NAME} == 'noname' ]]; then
```

Comment:
we set `wsrep_cluster_name` in` pxc-configure-pxc.sh ` maybe it is better to check it  in  https://github.com/percona/percona-xtradb-cluster-operator/blob/main/build/pxc-configure-pxc.sh#L41

Reply:
in the situation dns is missing，the pxc-configure-pxc.sh will not be executed ,and the cluster name will be "noname", which is the default name in the image. So, it should checked in the entrypoint.sh